packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -1151,7 +1152,24 @@ class ConversationService {
     this.clientID = clientID;
   }
 
-  public async toggleArchiveConversation(
+  public setConversationMutedStatus(
+    conversationId: string,
+    status: MutedStatus,
+    muteTimestamp: number | Date = new Date()
```

Comment:
This should never default to the current date. It has to be explicitly set to the latest known server timestamp in the conversation.

Reply:
Let's remove the default value then and continue.